File: 151002/Ravodin/DistributedComputing/Forum.Api/Models/Story.cs
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace Forum.Api.Models;

public class Story
{
    public long Id { get; set; }
    
    public string Title { get; set; }
    
    public string Content { get; set; }

    public DateTime Created { get; set; }

    public DateTime Modified  { get; set; }
    
    public Creator Creator { get; set; }
    
    
    public long CreatorId { get; set; }
    
    public List<Tag> Tags { get; set; }
}

File: 151003/Onuchina/DC/publicator/src/main/java/by/bsuir/entities/Story.java
package by.bsuir.entities;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
//@Table(name = "tbl_story", schema = "distcomp")
@Table(name = "tbl_story")
public class Story {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;
    @Column(name = "title")
    private String title;
    @Column(name = "content")
    private String content;
    @Column(name = "created")
    private Timestamp created;
    @Column(name = "modified")
    private Timestamp modified;
    @ManyToOne
    @JoinColumn(name = "author_id")
    private Author author;
    @ManyToMany
    @JoinTable(
            name = "tbl_story_tag",
            schema = "distcomp",
            joinColumns = @JoinColumn(name = "story_id"),
            inverseJoinColumns = @JoinColumn(name = "tag_id")
    )
    private List<Tag> tags;
}
